docs/related-tools/client-side-tools.md

```diff
@@ -2,10 +2,11 @@
 
 Client-side tools allow running any linter either locally or as part of your CI process and then integrating the results into your Codacy workflow. This way, Codacy will present the results coming from your linters alongside all the other code quality information in the dashboards.
 
-Currently, Codacy supports the client-side tools Clang-Tidy, Gosec, and Spotbugs. Read the guides to set up these client-side tools with Codacy:
+Currently, Codacy supports the client-side tools Clang-Tidy, Gosec, Staticcheck and Spotbugs. Read the guides to set up these client-side tools with Codacy:
```

Comment:
Very small style detail here, but I'm a fan of using the "Oxford comma" (which Grammarly and the Microsoft Manual of Style [also recommends](https://docs.microsoft.com/en-us/style-guide/punctuation/commas#use-a-comma)). :nerd_face: 

```suggestion
Currently, Codacy supports the client-side tools Clang-Tidy, Gosec, Staticcheck, and Spotbugs. Read the guides to set up these client-side tools with Codacy:
```